dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json

```diff
@@ -3716,12 +3716,12 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"} - ON(instance, task) dm_syncer_binlog_file{instance=~\"$instance\", task=~\"$task\", node=\"syncer\"}",
+          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"} - ON(instance) dm_syncer_binlog_file{instance=~\"$instance\", task=~\"$task\", node=\"syncer\"}",
```

Comment:
preview [link](http://172.16.5.177:3000/d/Ja68YqRnzxxxxxxx/dm-monitor-professional-fixed?editPanel=44&orgId=1&from=now-5m&to=now&var-task=All&var-source=mysql-replica-2-001&var-instance=172.16.5.177:8101&var-queueNo=All&var-type=All&var-stage=All&var-target_schema=All&var-target_table=All&var-interval=$__auto_interval_interval&var-worker=All)